mesonbuild/compilers.py

```diff
@@ -2434,6 +2454,9 @@ def thread_link_flags(self):
     def get_option_link_args(self, options):
         return []
 
+    def get_linker_allow_undefined(self):
+        return []
```

Comment:
That's only used for static linking, so probably not. Doesn't hurt to put it there though.